tests/unit/build_providers/lxd/test_lxd.py

```diff
@@ -188,7 +188,23 @@ def test_create(self):
         container.start_mock.assert_called_once_with(wait=True)
         self.assertThat(container.save_mock.call_count, Equals(2))
         self.assertThat(container.sync_mock.call_count, Equals(36))
-        self.assertThat(self.check_call_mock.call_count, Equals(29))
+        self.assertThat(self.check_output_mock.call_count, Equals(2))
```

Comment:
What's the second call? Why not just check against the full set of `self.check_output_mock.mock_calls`?

Reply:
it is the same

Reply:
OK. I still have trouble understanding the purpose of ever checking call_count whenever there are arguments that could be doing anything...